kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/TopicTests.java

```diff
@@ -45,15 +43,15 @@ public static void beforeAll() {
     @Test
     public void createTopic() {
         Topic topicToCreate = new Topic().setName("new-topic");
-        pages.open()
-                .goToSideMenu(CLUSTER_NAME, MainPage.SideMenuOptions.TOPICS);
-        pages.topicsList.pressCreateNewTopic()
+        topicsList.goTo(CLUSTER_NAME)
+                .waitUntilScreenReady()
+                .pressCreateNewTopic()
                 .setTopicName(topicToCreate.getName())
                 .sendData()
                 .waitUntilScreenReady();
-        pages.open()
-                .goToSideMenu(CLUSTER_NAME, MainPage.SideMenuOptions.TOPICS);
-        Assertions.assertTrue(pages.topicsList.isTopicVisible(topicToCreate.getName()),"isTopicVisible");
+        topicsList.goTo(CLUSTER_NAME)
```

Comment:
we don't need to remove navigation steps from test. please return mainPage.goTo()
.goToSideMenu(CLUSTER_NAME, bla-bla